authors/apps/profiles/urls.py

```diff
@@ -18,5 +19,8 @@
          name="following"),
     path("follow/followers", ProfileFollowersAPIView.as_view(),
          name="followers"),
-    path("", UserProfileListAPIView.as_view(), name="profile-list"),
+    path("", UserProfileListAPIView.as_view(),
+         name="profile-list"),
+    path('<username>/readstats', ReadStatsView.as_view(),
```

Comment:
Why do you need to specify a `username` if the user is already `logged in`, and from what I understand, a user can `only` get `their` `stats`?

Reply:
@Karuhanga As you might notice if you read the API spec, `GET /profiles/:username` would try to fetch you the profile of <username> hence the use of `GET /profiles/:username/readstats` to fetch the user's reading stats.

Reply:
True, I have noticed that. But in this case one cannot retrieve another user's reading stats the way they can retrieve their profile. In light of this therefore, I do not see the need to require the API consumer to supply a user for a user that is already logged in.

Reply:
@Karuhanga , thanks for the feedback, the issue has been resolved